block-storage/src/main/scala/coop/rchain/blockstorage/dag/InMemBlockDagStorage.scala

```diff
@@ -142,7 +141,7 @@ final class InMemBlockDagStorage[F[_]: Concurrent: Sync: Log](
                                                     s"Block ${Base16.encode(block.blockHash.toByteArray)} sender is empty"
                                                   ) >> newValidatorsLatestMessages.pure[F]
                                                 } else if (block.sender
-                                                             .size() == BlockHash.Length) {
+                                                             .size() == Validator.Length) {
```

Comment:
We had this bug, but I guess it's ok to fix it in feature branch and not allocate PR on dev for that?

Reply:
It's used in tests, how that is not failing now?

Reply:
My guess is that now everywhere in tests `ByteString.Empty` is used as a sender

Reply:
Actually we don't need this in memory DAG storage anymore. It leftover of file-based storage and with key-value based storage we can easily create in memory variant and use the same code in testing and production.

https://github.com/rchain/rchain/blob/2d5aac8a270ee8053240eb340db9bb3624b52372/casper/src/test/scala/coop/rchain/casper/engine/Setup.scala#L119-L121

Here is the PR to remove it #3426.

Reply:
We can leave this file as it is. It will be removed after sync with dev.